Project/Core/Configuration/AppConfig.cs

```diff
@@ -34,7 +34,11 @@ public static class AppConfig
         /// <summary>
         /// A path to a file with logs.
         /// </summary>
-        public static readonly string LOG_FILE = string.Join(DIRECTORY_SEPARATOR_STR, LOG_DIRECTORY, "log.txt");
+        public static readonly string LOG_FILE = string.Join(DIRECTORY_SEPARATOR_STR, LOG_DIRECTORY, "log.log");
+        /// <summary>
+        /// The size limit of log file in bytes.
+        /// </summary>
+        public static readonly long LOG_FILE_SIZE_LIMIT = 10 * 1024 * 1024; // 10 MB in bytes

```

Comment:
Hard to change. Write some converter function for this. Which will work so: GetSize(10, FileSize.MB)
or check if current exist.

Reply:
Okay